src/ontology/pato-edit.obo

```diff
@@ -20612,6 +20613,54 @@ synonym: "skin redness" EXACT []
 is_a: PATO:0000051 ! morphology
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: PATO:0040050
+name: diandrous
+def: "A phenotypic sex quality inhering in a  plant organism or a population of plants in which the flowers have two stamens." [http://orcid.org/0000-0002-2908-3327] {comment="https://orcid.org/0000-0001-8815-0078"}
+is_a: PATO:0001894 ! phenotypic sex
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040051
+name: monandrous
+def: "A phenotypic sex quality inhering in a  plant organism or a population of plants in which the flowers have only one stamen." [http://orcid.org/0000-0002-2908-3327, https://orcid.org/0000-0001-8815-0078]
+is_a: PATO:0001894 ! phenotypic sex
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040052
+name: protogynous
+def: "A sequential hermaphroditic quality whereby organisms start out as female and change to male at a later life stage." [https://en.wikipedia.org/wiki/Sequential_hermaphroditism] {comment="http://orcid.org/0000-0002-2908-3327"}
+synonym: "protogyny" EXACT []
+is_a: PATO:0070000 ! sequential hermaphrodite
+created_by: https://orcid.org/0000-0001-8815-0078
+
+[Term]
+id: PATO:0040053
```

Comment:
the ID for this term is PATO_0070038 in the OWL file

Reply:
What OWL file?

Reply:
What OWL file?

Reply:
the pato-edit.owl file

Reply:
Oh, I guess it is pato-edit.obo, that is what I meant, obo, not owl

Reply:
![image](https://user-images.githubusercontent.com/6722114/122827162-e001ac00-d298-11eb-80d1-5bed687de6b6.png)